README.md

```diff
@@ -14,29 +14,29 @@ First, add your AWS credentials to a new file called `.env`. You can see an exam
 ```bash
 git clone https://github.com/MAAP-Project/dashboard-api-starter.git
 cd dashboard-api-starter
-
-# Add your AWS credentials to a new file called `.env`. You can see an example of this file at `.env.example`.
-cp .env.example .env
 # Copy and configure the app
 cp stack/config.yml.example stack/config.yml
 ```
 
-**IMPORTANT:** Create if needed and ensure access to the buckets configured in `stack/config.yml`.
+Note, the local `stack/config.yml` file will only be used for running the app locally. Deployment to AWS is managed via CDK and github actions (See `.github/workflows/deploy.yml`).
+
+Datasets for `/v1/datasets` are loaded from a json file stored in S3 unless `ENV=local` is set when running the app. The S3 location for these datasets is defined by the `BUCKET` and `DATASET_METADATA_FILENAME` values in `stack/config.yml`: `s3://{BUCKET}/{DATASET_METADATA_FILENAME}`.
 
 ### Running the app locally
 
-To run the app locally, generate a config file and generate the static dataset json files.
+You can use `ENV=local` when running the app locally to use the `example-dataset-metadata.json` file as the source for `/v1/datasets`. This is useful for testing new dataset configurations.
```

Comment:
Do we have this set up?

Reply:
Yes? I've added this conditional for local file reading - https://github.com/MAAP-Project/dashboard-api-maap/blob/ba75b0406bfe175bc043deed2ca746229bd02c5c/dashboard_api/db/static/datasets/__init__.py#L36-L40

Reply:
ahh, sorry i didn't see that